README.md

```diff
@@ -6438,6 +6438,29 @@ config.prepareDatabase { db in
 let dbQueue = try DatabaseQueue(path: dbPath, configuration: config)
 ```
 
+#### Passphrase Datatype: String vs Data
```

Comment:
Your paragraph has no logical connection to the previous paragraphs:

- Managing the lifetime of the passphrase string (high-level)
- Managing the lifetime of the passphrase bytes (as low-level as it can be)

Your contribution, in order to be understood, and useful, must enrich the story. This is how it will help *everybody*, not just you.

Did you notice the "Managing the lifetime of the passphrase bytes" paragraph? It already contains everything you already need. For your contribution to be useful, you must add something.